lib/pact/command/cont_command.ex

```diff
@@ -91,7 +91,7 @@ defmodule Kadena.Pact.ContCommand do
     proof = Keyword.get(opts, :proof)
     rollback = Keyword.get(opts, :rollback, %Rollback{})
     keypairs = Keyword.get(opts, :keypairs, [])
-    signers = Keyword.get(opts, :signers, %SignersList{})
+    signers = Keyword.get(opts, :signers, [])
```

Comment:
@Odraxs @EdwinGuayacan  the default value in `new/2` for rollback is a struct, but `set_rollback/2` function does not match this but a boolean... so, this field will be a boolean or a struct?

Reply:
@miguelnietoa Because of this, I discovered that the values `step`, `pact_transaction_hash`, and `rollback` are defined as types even though they are actually just numeric, string, and boolean values, respectively.

I suppose it would be better if we change those methods to use their types?

Reply:
@Odraxs Mmmm given this situation, I think we can use the type structs internally, but the commands could receive the primitive value in _number, string, boolean_, as you mentioned...

I think it can follow the same logic behind `set_network/2`

Is it clear? What do you think?

Reply:
Yes, that's the idea, in fact the primitive data is received and the types are created in `Cont Payload` type